.gitignore

```diff
@@ -25,6 +25,7 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+.idea/
```

Comment:
The `.idea/` directory is already listed here, and it appears again at line 162. You can remove the duplicate entry at line 162 to avoid redundancy.